<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgramSettingsDialog</class>
 <widget class="QDialog" name="ProgramSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Program Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gbSettingsMain">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#gbSettingsMain {
     border: 1px solid gray;
     margin-top: 0.5em;
}
QGroupBox#gbSettingsMain::title { 
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
     </property>
     <property name="title">
      <string>Data Refresh Rate Settings</string>
     </property>
     <widget class="QComboBox" name="cbMarketSampleRate">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>46</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>20</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update bets every :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="cbBetsSampleRate">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>21</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>45</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update prices every :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBetPlacementSettings">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#gbBetPlacementSettings {
     border: 1px solid gray;
     margin-top: 0.5em;
}
QGroupBox#gbBetPlacementSettings::title { 
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
     </property>
     <property name="title">
      <string>Bet Placement Settings</string>
     </property>
     <widget class="QCheckBox" name="chkBetPersistenceFlag">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Placed Bets Persist At In-Play</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbAutoPlaceOffsetBet">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>118</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkAutoPlaceOffsetBet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>116</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Automatically place an opposing bet at a</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>119</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>tick offset from placed bets.</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkLadderBettingEnabled">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Enable Ladder Betting</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkInPlayViewBettingEnabled">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Enable In-play View Betting</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chkGridViewBettingEnabled">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>    Enable Grid View Betting</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbLadderSettings">
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox#gbLadderSettings {
     border: 1px solid gray;
     margin-top: 0.5em;
}
QGroupBox#gbLadderSettings::title { 
     subcontrol-origin: margin;
     left: 10px;
     padding: 0 3px 0 3px;
 }</string>
     </property>
     <property name="title">
      <string>Ladder Interface Settings</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>29</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ladder WOM Display Depth : </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="cbWOMDepth">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>22</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="sbLadderHistInterval1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>55</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>599</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblChooseRunner_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>55</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lower Traded Volume Histogram Interval :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblChooseRunner_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Upper Traded Volume Histogram Interval :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbLadderHistInterval2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLowerHistCol">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>55</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Lower Traded Histogram Colour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpperHistCol">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>80</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Upper Traded Histogram Colour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTotalVolHistCol">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>150</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Total Traded Histogram Colour</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveSettings">
       <property name="font">
        <font>
         <family>Ubuntu</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save And Apply Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
